### PR TITLE
AOM-100: Add a sticky Navbar.

### PR DESCRIPTION
**Pull request by justmesam**
## JIRA TICKET NAME:
[AOM-100: Add a sticky Navbar](https://issues.openmrs.org/browse/AOM-100)

#### Currently:
When you scroll down through the list of the add-ons you don't have access to the nav bar or an easy way to scroll back to the top of the page. 

#### After-Fix:
This ticket makes the navbar fixed to the top of the page and adds a floating button that which when the user clicks it takes the page goes back to the top of the page.

### CODE REVIEW

File: app/css/openmrs-addonmanager.css [hunk 1]
```diff
@@ -803,7 +803,15 @@ body.loading .waiting-modal {
 }
 
 .install {
-    margin-top: 5px 
+    margin-top: 5px
+}
```
Comment (Charpell) [inline app/css/openmrs-addonmanager.css]: Only affect changes to the code related to this PR.

**Comment by brayoh** [comment]
Looks good to me.

File: app/js/components/common/Header.jsx [hunk 2]
```diff
@@ -196,7 +196,7 @@ export default class Header extends Component {
 
   render() {
     return (
-      <div>
+      <div style={{...this.props.style, zIndex: 1}}>
```
Comment (kwahalf) [inline app/js/components/common/Header.jsx:199]: which styles are you passing via  props?

Comment (justmesam) [reply app/js/components/common/Header.jsx:199]: This are the styles passed from the high order component `Sticky` which are used to make the header component sticky to the top of the page.

**Review by aenkya — COMMENTED** [review]
LGTM

File: tests/components/App.test.js [hunk 3]
```diff
@@ -12,13 +12,14 @@ import React from 'react';
 import { mount, shallow } from 'enzyme';
 import { expect } from 'chai';
 import sinon from 'sinon';
+import { StickyContainer, Sticky } from 'react-sticky';
 import App from '../../app/js/components/App.jsx';
 import Header from '../../app/js/components/common/Header';
 
 describe('<App />', () => {
 
     it('should mount the Header in itself', () => {
```
Comment (LWanjiru) [inline tests/components/App.test.js]: `Should mount the header in itself`?... What does this statement mean?

**Review by LWanjiru — COMMENTED** [review]
Looks good.

**Comment by patlub** [comment]
Looks Good

**Comment by malmike** [comment]
Looks Good

**Comment by Annettesunday** [comment]
Looks good 👍

File: app/js/components/common/Header.jsx (hunk 2 above)
Comment (kwahalf) [reply app/js/components/common/Header.jsx:199]: Okay looks good

**Review by kwahalf — APPROVED** [review]
good work

**Pull request merged**